packages/connectivity/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.4.2+2
+
+* Add test driver.
```

Comment:
I would say "Added integration test." Although we're using Flutter driver to launch the test, we don't use the FlutterDriver API to do anything, so it may be confusing otherwise.